aviatrix/resource_aviatrix_splunk_logging.go

```diff
@@ -0,0 +1,146 @@
+package aviatrix
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-aviatrix/goaviatrix"
+)
+
+func resourceAviatrixSplunkLogging() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAviatrixSplunkLoggingCreate,
+		Read:   resourceAviatrixSplunkLoggingRead,
+		Update: resourceAviatrixSplunkLoggingUpdate,
+		Delete: resourceAviatrixSplunkLoggingDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"server": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Server IP",
+			},
+			"port": {
```

Comment:
What is the purpose of having a default value of `-1`?

Reply:
rest api will return err when port is 0, so I modified the code to simply check if the port is 0 or not.